lib/elasticity/index_config.rb

```diff
@@ -29,7 +29,7 @@ def client
     def definition
       return @definition if defined?(@definition)
       @definition = {
-        settings: merge_settings, mappings: { @document_type => @mapping || {} }
+        settings: merge_settings, mappings: { @document_type => @mapping.deep_stringify_keys || {} }
```

Comment:
without `stringifying`,  models on the client side (Doximity) that use symbols for keys will raise an error when
`remap!`  is run
```
NoMethodError: undefined method `keys' for nil:NilClass
from /Users/david/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/es-elasticity-0.10.0/lib/elasticity/strategies/alias_index.rb:70:in `block in remap'
```